package.json

```diff
@@ -62,14 +62,16 @@
     "start": {
       "command": "babel-node bin/server",
       "env": {
-        "DEBUG": "app:*"
+        "DEBUG": "app:*",
+        "CONTROLLER_SERVICE": "https://dev-logistics-wizard-controller.mybluemix.net/api/v1"
```

Comment:
this points to our live demo. In our chat yesterday we said this should not be the case but rather point to 0.0.0.0:5000 which is the local configuration defined in https://github.com/IBM-Bluemix/logistics-wizard/wiki/Quick-Start